acceptance/tests/parser_functions/hiera_in_templates.rb

```diff
@@ -2,7 +2,7 @@
 
 @module_name = "hieratest"
 @coderoot = master.tmpdir("#{@module_name}")
-@resultdir = agent.tmpdir("#{@module_name}_results")
+@resultdir = agents.first.tmpdir("#{@module_name}_results")
```

Comment:
I don't think this will work as you might have an posix agent on the master, and a windows agent, and there's no guarantee about which agent first will return, and the resulting path likely won't work for both.